tests/foreman/api/test_contentview.py

```diff
@@ -47,6 +46,7 @@
 from robottelo.constants import REPOSET
 from robottelo.datafactory import invalid_names_list
 from robottelo.datafactory import valid_data_list
+from robottelo.decorators import fixture
```

Comment:
I really don't like using this import, when `pytest` is already imported. There's nothing unique, its just a local assignment in the decorators module.

Reply:
Same here, Just left out from my eyes filter :P